src/plugins/intel_cpu/tests/unit/graph/merge_transpose_reorder_test.cpp

```diff
@@ -173,32 +183,33 @@ class MergeTransposeReorderCPUTest : public testing::WithParamInterface<MergeTra
 
     std::shared_ptr<GraphContext> m_context;
     std::unique_ptr<Graph> m_graph;
+    std::shared_ptr<NetworkMemoryControl> networkMemoryControl = std::make_shared<NetworkMemoryControl>();
```

Comment:
It seems that the variable is not used. Could you please revise all the other similar places?

Reply:
Cleaned up